### PR TITLE
docs: update the testing guide

### PR DESCRIPTION
**Pull request by elopio**

### CODE REVIEW

**Comment by elopio** [comment]
@jcarpanelli adding you as a reviewer because I saw you made the PR renaming to TestHelper. Please let me know if this looks good.

File: packages/docs/docs/docs/testing.md [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
 
-The `zos` package provides a `TestApp()` function to retrieve your application structure from the `zos.json` file and deploy everything to the current test network. All contracts you have registered via `zos add`, plus all the contracts provided by the EVM package you have linked, will be available. The returned [`App`](https://github.com/zeppelinos/zos-lib/blob/master/src/app/App.js) object provides convenient methods for creating upgradeable instances of your contracts, which you can use for testing.
+The `zos` package provides a `TestApp()` function to retrieve your project structure from the `zos.json` file and deploy everything to the current test network. All contracts you have registered via `zos add`, plus all the contracts provided by the EVM packages you have linked, will be available. The returned [`App`](https://github.com/zeppelinos/zos-lib/blob/master/src/app/App.js) object provides convenient methods for creating upgradeable instances of your contracts, which you can use for testing.
```
Comment (jcarpanelli) [inline packages/docs/docs/docs/testing.md]: This should be `TestHelper()` instead of `TestApp()`.

File: packages/docs/docs/docs/testing.md [hunk 2]
```diff
@@ -1,18 +1,19 @@
...
 
 > **Important:** for `TestApp` to work correctly in your testing environment, you need to set the `NODE_ENV` environment variable to `test` when running your tests. For instance, if you are using truffle, run `NODE_ENV=test truffle test`.
```
Comment (jcarpanelli) [inline packages/docs/docs/docs/testing.md]: `TestHelper` here too.

File: packages/docs/docs/docs/testing.md [hunk 3]
```diff
@@ -22,21 +23,21 @@ Given a small application, with a `Sample` contract, linked to an EVM package th
...
   "stdlib": {
-    "name": "openzeppelin-zos",
-    "version": "1.9.0"
+    "name": "openzeppelin-eth",
+    "version": "2.0.2"
   }
 }
 ```
 
 To set up the full application in a test file, first import `TestApp` from `zos`:
```
Comment (jcarpanelli) [inline packages/docs/docs/docs/testing.md]: `TestHelper`

File: packages/docs/docs/docs/testing.md [hunk 4]
```diff
@@ -22,21 +23,21 @@ Given a small application, with a `Sample` contract, linked to an EVM package th
     "Sample": "Sample"
   },
   "stdlib": {
-    "name": "openzeppelin-zos",
-    "version": "1.9.0"
+    "name": "openzeppelin-eth",
+    "version": "2.0.2"
```
Comment (facuspagnuolo) [inline packages/docs/docs/docs/testing.md]: this json should be entirely updated to:
```json
{
  "zosversion": "2",
  "name": "my-project",
  "version": "0.1.0",
  "contracts": {
    "Sample": "Sample"
  },
  "dependencies": {
    "openzeppelin-eth": "2.0.2"
  }
```

File: packages/docs/docs/docs/testing.md (hunk 1 above)
Comment (spalladino) [reply packages/docs/docs/docs/testing.md]: > The returned [`App`](https://github.com/zeppelinos/zos-lib/blob/master/src/app/App.js) object

`TestHelper` returns a `Project` (either a `SimpleProject` or an `AppProject`), not an `App`.

File: packages/docs/docs/docs/testing.md [hunk 5]
```diff
@@ -53,15 +54,15 @@ it('should create a proxy', async function () {
...
 
 const Sample = artifacts.require('Sample')
-const StandardToken = artifacts.require('StandardToken')
+const ERC20 = artifacts.require('ERC20')
 
 contract('Sample', function ([_, owner]) {
 
   beforeEach(async function () {
-    this.app = await TestApp({ from: owner })
+    this.app = await TestHelper({ from: owner })
```
Comment (spalladino) [inline packages/docs/docs/docs/testing.md]:
```suggestion
    this.project = await TestHelper({ from: owner })
```

File: packages/docs/docs/docs/testing.md [hunk 6]
```diff
@@ -71,7 +72,7 @@ contract('Sample', function ([_, owner]) {
   })
 
   it('should create a proxy for the EVM package', async function () {
-    const proxy = await this.app.createProxy(StandardToken);
+    const proxy = await this.app.createProxy(ERC20);
```
Comment (spalladino) [inline packages/docs/docs/docs/testing.md]:
```suggestion
    const proxy = await this.project.createProxy(ERC20, { contractName: 'StandaloneERC20', packageName: 'openzeppelin-eth' });
```

**Comment by elopio** [comment]
damn, I did a terrible job here :man_facepalming: 
Thanks to all of you for catching the mistakes, I hope it's better now.

**Pull request merged**